integration-tests/tests/find-workflow.js

```diff
@@ -1,12 +1,13 @@
 const firecloud = require('../utils/firecloud-utils')
 const { billingProject, testUrl, workflowName } = require('../utils/integration-config')
 const { withWorkspace } = require('../utils/integration-helpers')
-const { click, clickable, findElement, findText, signIntoTerra } = require('../utils/integration-utils')
+const { click, clickable, dismissNotifications, findElement, findText, signIntoTerra } = require('../utils/integration-utils')
 
 
 const testFindWorkflowFn = withWorkspace(async ({ page, workspaceName }) => {
   await page.goto(testUrl)
   await signIntoTerra(page)
+  await dismissNotifications(page)
```

Comment:
Why not call `dismissNotifications` from `signIntoTerra`?

Reply:
We made this decision because there are lots of sign ins just for Ajax, like when creating or deleting a workspce

Reply:
Good reason. 👍